app/data/toys.json

```diff
@@ -39,6 +39,16 @@
             "icon": "inv_elemental_primal_water",
             "itemId": 168016,
             "name": "Hyper-Compressed Ocean"
+          },
+          {
+            "icon": "spell_nzinsanity_chasedbyshadows",
+            "itemId": 174830,
+            "name": "Shadowy Disguise"
```

Comment:
This is not specific to this expansion. Should go in the Pet Battles category, not BFA.

Reply:
Fixed.